chapters/en/module1.md

```diff
@@ -16,12 +16,33 @@ id: 1
 <slides source="module1/module1_01" shot="1" start="0:003" end="05:31"> </slides>
 </exercise>
 
-<exercise id="2" title="Test Your Knowledge">
+<exercise id="2" title="❓ Test Your Knowledge">
```

Comment:
I tested adding a question mark with a space here and one without in the next question. Do you think this looks good or should we try another emoji?